litefs/cloud-restore.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Restoring from a LiteFS Cloud Backup
+layout: docs
+sitemap: false
+nav: litefs
+toc: true
+---
```

Comment:
We'll need to double-check this section once @chrismccord implements the Restore UI.